config.go

```diff
@@ -1,267 +1,268 @@
+	pushCmdLineItem("-p", "Forces use specified password", true, false, funcDBPass, &res)
+	pushCmdLineItem("-d", "Forces use specified DBHost", true, false, funcDBHost, &res)
+	pushCmdLineItem("-p", "Forces use specified DBPort", true, false, funcDBPort, &res)
+	pushCmdLineItem("-m", "Sets display Mode 2=subsystems,1=all", true, false, funcMode, &res)
+	pushCmdLineItem("-h", "This Help", false, false, funcHelp, &res)
 
 	return res
 }
 
-func func_help		(conf *configuration,fn []string)		(error){
-	return errors.New("Command Help")
+func funcHelp(conf *configuration, fn []string) error {
+	return errors.New("command Help")
 }
 
-func func_outtype(conf *configuration, jout []string)			(error){
-	(*conf).Jout=jout[0]
+func funcOutType(conf *configuration, JOut []string) error {
+	(*conf).Jout = JOut[0]
 	return nil
 }
 
-func func_jconf		(conf *configuration,fn []string)		(error){
+func funcJconf(conf *configuration, fn []string) error {
 	jsonFile, err := os.Open(fn[0])
 	if err != nil {
 		return err
-		}
+	}
 	byteValue, _ := ioutil.ReadAll(jsonFile)
 	jsonFile.Close()
-	err=json.Unmarshal(byteValue, conf)
+	err = json.Unmarshal(byteValue, conf)
 	if err != nil {
 		return err
-		}
+	}
 	return nil
 }
 
-func func_symbol	(conf *configuration, fn []string)	(error){
-	(*conf).Symbol=fn[0]
+func funcSymbol(conf *configuration, fn []string) error {
+	(*conf).Symbol = fn[0]
 	return nil
 }
 
-func func_DBUser	(conf *configuration, user []string)	(error){
-	(*conf).DBUser=user[0]
+func funcDBUser(conf *configuration, user []string) error {
+	(*conf).DBUser = user[0]
 	return nil
 }
 
-func func_DBPass	(conf *configuration, pass []string)	(error){
-	(*conf).DBPassword=pass[0]
+func funcDBPass(conf *configuration, pass []string) error {
+	(*conf).DBPassword = pass[0]
 	return nil
 }
 
-func func_DBHost	(conf *configuration, host []string)	(error){
-	(*conf).DBURL=host[0]
+func funcDBHost(conf *configuration, host []string) error {
+	(*conf).DBURL = host[0]
 	return nil
 }
 
-func func_DBPort	(conf *configuration, port []string)	(error){
+func funcDBPort(conf *configuration, port []string) error {
 	s, err := strconv.Atoi(port[0])
-	if err!=nil {
+	if err != nil {
 		return err
-		}
-	(*conf).DBPort=s
+	}
+	(*conf).DBPort = s
 	return nil
 }
 
-func func_instance	(conf *configuration, instance []string)    (error){
+func funcInstance(conf *configuration, instance []string) error {
 	s, err := strconv.Atoi(instance[0])
-	if err!=nil {
+	if err != nil {
 		return err
-		}
-	(*conf).Instance=s
+	}
+	(*conf).Instance = s
 	return nil
 }
 
-func func_Mode		(conf *configuration, mode []string)    (error){
+func funcMode(conf *configuration, mode []string) error {
 	s, err := strconv.Atoi(mode[0])
-	if err!=nil {
+	if err != nil {
 		return err
-		}
-	if s<1 || s>2 {
-		return errors.New("unsupported mode")
-		}
-	(*conf).Mode=s
+	}
+	if s < 1 || s > 2 {
+		return errors.New("unsupported Mode")
+	}
```

Comment:
Same as before, why change also the message string case?
Is it because you want to match the message string with the variable field name?